### DISCUSSION TITLE
The new Quasar docs - fonts are terribly bad

### DISCUSSION THREAD

**Discussion by tomaszwaraksa** [quasarframework/quasar#15475] [Ideas / Proposals] [answered]
Folks, 

About https://new-docs.quasar.dev/docs

Documentation is not a graphic design project. Primary function of it is to provide searchable information in least bothersome way. Of crucial importance is legibility - because many in our profession struggle with eyesight deficiencies and strained eyes.

The current choice of fonts is ... questionable, to put it mildly.

Character size of the left menu is too small, while spacing is way too big. It just feels off. 

Search results - it's so hard to make any sense of it. I don't know... maybe this qualifies as cute, or excellent design, but from the point of view of a guy with thick glasses spending 10+ hours a day behind the keyboard, it's torture. Plenty of vertical lines and boxes. Poor contrasts. And again, font feels squashed vertically, while too stretched horizontally due to poor choice of character spacing. 

Last but not least, all fonts seem fuzzy. Compare with the old ones, which are crisp. It's straining the eyes a lot.

![image](https://user-images.githubusercontent.com/1721566/220631766-fdcdb818-1327-46c3-962c-80ecf391d6cc.png)

![image](https://user-images.githubusercontent.com/1721566/220632449-954bec7a-5fb5-48c9-9666-cf3c260c4fa2.png)

See it folks and tell me what you think of it.

**Comment by rbeeli** [comment]
Fully agree.

There is not enough contrast either in light mode. Gray on light-gray, really? Please get rid of the gray background, it looks dated and does not help readability. IMO default theme (light/dark) should be the current browser/system theme, or light, not the dark theme, as many others follow the same practice.

And why is the font not the same as the font used in the actual framework? This inconsistency is neither confidence inspiring, nor visually pleasing to look at.

**Comment by rstoenescu** [comment, marked as answer]
Hi All,

Thanks for the feedback. However, please let's keep the discussion in one place (in the initial thread): https://github.com/quasarframework/quasar/discussions/15168
